packages/select/tests/Select.test.js

```diff
@@ -160,6 +160,86 @@ describe('Select', () => {
     });
   });
 
+  test('select all submits from null initial value', async () => {
+    const onSubmit = jest.fn();
+    const { container, getByText } = render(
+      <Form
+        initialValues={{
+          selectAll: null,
+        }}
+        onSubmit={onSubmit}
+        validationSchema={multiValueSchema('selectAll', true, 1, 4)}
+      >
+        <Select
+          name="selectAll"
+          isMulti
+          options={options}
+          data-testid="single-select"
+          allowSelectAll
+        />
+        <Button type="submit">Submit</Button>
+      </Form>
+    );
+
+    await selectItem(container, getByText, 'Select all');
+
+    await fireEvent.click(getByText('Submit'));
+
+    await wait(() => {
```

Comment:
`wait` is already defined in this file so I think this will be fine when it gets merged..

Reply:
Figured out the issue here. Should be good to go now.